README.md

```diff
@@ -44,29 +56,11 @@ root folder to start with this option:
     yarn install
     run-e2e.sh -l
 
-## Running only the backend
+### Running only the backend
```

Comment:
As we discussed, we could also here point more out, that this can be used to play with the background